enterprise/causal-clustering/src/main/java/org/neo4j/causalclustering/core/consensus/RaftMessages.java

```diff
@@ -37,6 +37,26 @@
 
 public interface RaftMessages
 {
+
+    interface Visitor<T, E extends Exception>
```

Comment:
Maybe call this a Handler or MessageHandler? And the respective handler methods handle( .. )?

Reply:
I'll rename to `Handler`. The outer class name gives context as to what it is handling.